### PR TITLE
Update conflict alert 

### PR DESCRIPTION
**Pull request by kktai1512**
https://trello.com/c/yXxVtC22/27-update-conflict-alert-component

CONFLICT: 'There are conflicts in your booking!',
 EMPTY_FIELD: 'Oops! There seems to be an empty field.',
 SERVER_ERROR: 'Server is down now please try again later',

add 3 types of conflict to the alert

### CODE REVIEW

**Comment by kktai1512** [comment]
[Update conflict alert component](https://trello.com/c/yXxVtC22/27-update-conflict-alert-component)

File: frontend/src/components/ConflictAlert.tsx [hunk 1]
```diff
@@ -33,19 +33,33 @@ const ViewText = styled.a`
...
+type Props = {
+  errorType: 'CONFLICT' | 'EMPTY_FIELD' | 'SERVER_ERROR'
+}
+interface ErrorMessage {
+  [key: string]: string
+}
+const errorMessage: ErrorMessage = {
+  CONFLICT: 'There are conflicts in your booking!',
+  EMPTY_FIELD: 'Oops! There seems to be an empty field.',
+  SERVER_ERROR: 'Server is down now please try again later',
```
Comment (jason-ng-zq99) [inline frontend/src/components/ConflictAlert.tsx:45]: I think you should include the full stop in this message too!

**Review by jason-ng-zq99 — CHANGES_REQUESTED** [review]
Minor typo that I think needs to be changed, but otherwise LGTM! 😄

File: frontend/src/components/ConflictAlert.tsx (hunk 1 above)
Comment (kktai1512) [reply frontend/src/components/ConflictAlert.tsx]: wah good catch! thanksss

Comment (jason-ng-zq99) [reply frontend/src/components/ConflictAlert.tsx]: 👍🏼

**Pull request merged**